docs/docsource/reference/access_controls/bucket_policies/index.rst

```diff
@@ -0,0 +1,21 @@
+Bucket Policy
+=============
+
+Bucket policies afford you the opportunity to restrict or permit access to
+buckets based on policies owned by the account that owns a bucket. Account users
+can create policies for specific buckets and then grant other accounts access
+permissions to the same bucket (and all objects in it).
+
+User accounts can create a bucket policy for any bucket they own, and can grant
+bucket access to other accounts. Once bucket access is granted, all accounts
+with shared access to the bucket can access all objects in the bucket. Bucket
```

Comment:
This isn't true. Bucket policies allow access only to the specific actions listed in them, so for example it's possible for one user to be granted access only a GetBucketLocation action, while another user could be granted GET access to any resource, while another user has access to everything.
We can discuss in person if needed :)